pom.xml

```diff
@@ -9,7 +9,7 @@
     <description>Simple Timer for Vaadin Flow</description>
 
     <properties>
-        <vaadin.version>14.0.12</vaadin.version>
```

Comment:
Upgrading vaadin.version is unrelated to FlowingCode/AddonsDemoV14#3 
Please commit separately